src/charts/CytoViz/CytoViz.js

```diff
@@ -128,7 +157,24 @@ export const CytoViz = (props) => {
       });
     }
   };
-
+  /**
+   * appends meta data needed for the visualisatation to the data field of the edges
+   * @param {*} edges in the Cityscape format
+   * @returns the elements object with fields: {asInEdgeHighlighted: false, _asOutEdgeHighlight: false}
+   */
+  const appendMissingDataValues = (edges) => {
```

Comment:
I'm wondering if we really need this function (it may take some time to process large graph instances :slightly_smiling_face:).

IIUC this function runs through all edges of the graph, and set the data `asInEdgeHighlighted` and `asOutEdgeHighlighted` to false if these keys did not exist. Yet we may not need to set all these values explicitly to false, because we're using selectors like `edge[?asInEdgeHighlighted]` in the styles and cytoscape documentation states:
> [?name] Matches elements if the specified data attribute is a [truthy](http://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/) value (e.g. [?foo]).

**Undefined values should be considered as false**, we should thus not need the function `appendMissingDataValues` for `asInEdgeHighlighted` and `asOutEdgeHighlighted` values :smile: